test/models/collection.test.ts

```diff
@@ -1,51 +1,54 @@
 // // TODO: Add collection type association
 
-// import app from '../../src/app'
-
-// describe('CRUD operation on \'Collection\' model', () => {
-//   const model = app.service('collection').Model
-
-//   before(async () => {
-//     setTimeout(() => {
-//       console.log('Waited for thirty seconds before test started.')
-//     }, 30000)
-//   })
-
-//   it('Create', done => {
-//     model.create({
-//       name: 'test',
-//       description: 'test description',
-//       metadata: JSON.stringify({ a: 'test' }),
-//       isPublic: true
-//     }).then(res => {
-//       done()
-//     }).catch(done)
-//   })
-
-//   it('Read', done => {
-//     model.findOne({
-//       where: {
-//         name: 'test'
-//       }
-//     }).then(res => {
-//       done()
-//     }).catch(done)
-//   })
-
-//   it('Update', done => {
-//     model.update(
-//       { description: 'test1 description' },
-//       { where: { name: 'test' } }
-//     ).then(res => {
-//       done()
-//     }).catch(done)
-//   })
-
-//   it('Delete', done => {
-//     model.destroy({
-//       where: { name: 'test' }
-//     }).then(res => {
-//       done()
-//     }).catch(done)
-//   })
-// })
+import app from '../../src/app'
+import { sequelize } from '../../src/models'
+
+describe('CRUD operation on \'Collection\' model', () => {
+  const model = app.service('collection').Model
+
+  before(async () => {
+    await sequelize.sync()
```

Comment:
We will not want to do this operation before every test file. For the moment, we can just run it once, before the very first set of integration tests. (In this case, it's the Collection tests, since that's the only one I uncommented.)